.travis/stages/verify/unit-testing/run

```diff
@@ -3,10 +3,8 @@
 set -eEux
 
 # Run Tests
-./gradlew --parallel test
```

Comment:
Does it work if `--parallel` just is written `-parallel` now?

Reply:
FWIW, from the gradle help text:
> --parallel                Build projects in parallel. Gradle will attempt to determine the optimal number of executor threads to use.

A pseudo general convention is flags that are fully spelled out get double dashes with single dash word delimiters, eg: `--first-second-word`, while 'short flags' that are single letters are single dash, eg: `-p`